src/DynamoCore/Core/Threading/UpdateRenderPackageAsyncTask.cs

```diff
@@ -184,7 +183,7 @@ private void GetRenderPackagesFromMirrorData(MirrorData mirrorData, ref List<str
                     if (solid != null)
                     {
                         foreach (var geom in solid.Edges.Select(edge => edge.CurveGeometry)) {
-                            geom.Tessellate(package, -1.0, maxTesselationDivisions);
+                            geom.Tessellate(package, -1.0, factory.MaxTessellationDivisions);
                             geom.Dispose();
```

Comment:
This is out of scope, but it doesn't seem like we use that first parameter anywhere.  Maybe we should just remove it...